frontend/packages/dev-console/src/components/formik-fields/ResourceLimitField.tsx

```diff
@@ -0,0 +1,59 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import cx from 'classnames';
+import { useField, useFormikContext, FormikValues } from 'formik';
+import { FormGroup, ControlLabel, HelpBlock } from 'patternfly-react';
+import { NumberSpinner, Dropdown } from '@console/internal/components/utils';
+import { getValidationState } from './field-utils';
+import { ResourceLimitFieldProps } from './field-types';
+
+const ResourceLimitField: React.FC<ResourceLimitFieldProps> = ({
+  inputLabel,
+  unitLabel,
+  name,
+  unitName,
+  helpText,
+  ...props
+}) => {
+  const [field, { touched, error }] = useField(name);
+  const { setFieldValue, setFieldTouched } = useFormikContext<FormikValues>();
+
+  return (
+    <FormGroup controlId={`${name}-field`} validationState={getValidationState(error, touched)}>
```

Comment:
This feels more like a normal component that combines two formik fields and not a formik field itself. Might be better to move it inside `import/resource-limits` or something like that and treat it as a normal component. What do you think?